src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -80,19 +80,20 @@ def run(self, revision):
         logger.info('Modified files', files=modified_files)
 
         # mach configure
-        logger.info('Mach configure ...')
+        logger.info('Mach configure…')
```

Comment:
I agree with @garbas 
This may have unknown effects, and does not really add anything IMHO.